lib/ansible/modules/system/iptables.py

```diff
@@ -460,6 +460,19 @@
     limit_burst: 20
     log_prefix: "IPTABLES:INFO: "
     log_level: info
+
+- name: Block container to visit 169.254.169.254
+  iptables:
+    action: insert
+    comment: prevent container to visit aws meta server
+    chain: FORWARD
+    protocol: tcp
+    destination_port: 80:443
+    destination: 169.254.169.254
+    in_interface: br-+
```

Comment:
This deserves more information. What is `br-+`, and how is that related to containers?

Reply:
Sure,
br-+ is docker network names in bridge mode start with "br-", this is the docker network containers are using.